R/format_PEW_temp.R

```diff
@@ -0,0 +1,419 @@
+#'
+#'
+#' #### Exclude not breeding data
+#' # filter(Method != "Night Control Winter")
+#'
+#'
+#'
+#' message("Compiling brood information...")
+#' Brood_data <- create_brood_PEW(data)
+#'
+#'
+#' create_brood_PEW <- function(data){
+#'
+#'
+#'
+#'
+#'
+#'
+#'   # #### Final arrangement
+#'   # dplyr::select(BroodID, PopID, BreedingSeason, Species, Plot, LocationID,
+#'   #               FemaleID, MaleID, ClutchType_observed, ClutchType_calculated,
+#'   #               LayDate_observed, LayDate_min, LayDate_max,
+#'   #               ClutchSize_observed, ClutchSize_min, ClutchSize_max,
+#'   #               HatchDate_observed, HatchDate_min, HatchDate_max,
+#'   #               BroodSize_observed, BroodSize_min, BroodSize_max,
+#'   #               FledgeDate_observed, FledgeDate_min, FledgeDate_max,
+#'   #               NumberFledged_observed, NumberFledged_min, NumberFledged_max,
+#'   #               AvgEggMass, NumberEggs, AvgChickMass, NumberChicksMass,
+#'   #               AvgTarsus, NumberChicksTarsus, OriginalTarsusMethod, ExperimentID)
+#'
+#'
+#'
+#' }
+#'
+#'
+#'
+#'
+#'
+#' #### VARIABLES
+#' # cat(Brood_data_names, sep="\n")
+#' # BroodID
+#' # PopID
+#' # BreedingSeason
+#' # Species
+#' # Plot
+#' # LocationID
+#' # FemaleID
+#' # MaleID
+#' # ClutchType_observed
+#' # ClutchType_calculated
+#' # LayDate_observed
+#' # LayDate_min
+#' # LayDate_max
+#' # ClutchSize_observed
+#' # ClutchSize_min
+#' # ClutchSize_max
+#' # HatchDate_observed
+#' # HatchDate_min
+#' # HatchDate_max
+#' # BroodSize_observed
+#' # BroodSize_min
+#' # BroodSize_max
+#' # FledgeDate_observed
+#' # FledgeDate_min
+#' # FledgeDate_max
+#' # NumberFledged_observed
+#' # NumberFledged_min
+#' # NumberFledged_max
+#' # AvgEggMass NumberEggs
+#' # AvgChickMass
+#' # NumberChicksMass
+#' # AvgTarsus
+#' # NumberChicksTarsus
+#' # OriginalTarsusMethod
+#' # ExperimentID
+#'
+#'
+#'
+#' #### --------------------------------------~
+#' #### LOCATION DATA                      #### done, check
+#' #### --------------------------------------~
+#'
+#' #### There is no information about coordinates of nestboxes >> use the same of location ?
+#' #### LocationID >> assume is the same as LocationID ?
+#'
+#' message("Compiling location information...")
+#' Location_data <- create_location_PEW(pew_data)
+#'
+#'
+#' create_location_PEW <- function(data){
+#'
+#'   Location_data <-
+#'     data %>%
+#'     dplyr::select(BreedingSeason, Date, NestboxID, PopID) %>%
+#'     group_by(NestboxID) %>%
+#'     arrange(BreedingSeason, Date) %>%
+#'     dplyr::mutate(StartSeason = first(BreedingSeason),
+#'                   EndSeason = last(BreedingSeason)) %>%
+#'     select(-BreedingSeason, -Date) %>%
+#'     dplyr::distinct() %>%
+#'     dplyr::ungroup() %>%
+#'     dplyr::mutate(LocationID = NestboxID, # CHECK
+#'                   LocationType = "NB",
+#'                   Latitude  = 51.266667,
+#'                   Longitude = 4.466667,
```

Comment:
Ask data owner if they have this info (they often do but just don't share it 😓).